### PR TITLE
Zoom implementation for HTML5

### PR DESCRIPTION
**Pull request by Tainan404**
This PR applies the zoom feature for html5:
 - The zoom toolbar.
 - mouse wheel events for zoom in/out.

Missing features:
 - ~Zoom in touch events.~
 - ~Hand navigation.~

### CODE REVIEW

File: bigbluebutton-html5/private/config/settings-development.json [hunk 1]
```diff
@@ -140,7 +140,8 @@
           "moveCursor",
           "sendAnnotation",
           "removePresentation",
-          "setPresentation"
+          "setPresentation",
+          "zoomSlide"
```
Comment (capilkey) [inline bigbluebutton-html5/private/config/settings-development.json]: Need to add this settings change to the production file also

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-overlay/component.jsx [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import SlideCalcUtil from '/imports/utils/slideCalcUtils';
 
 const CURSOR_INTERVAL = 16;
+const MYSTERY_NUM = 2;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-overlay/component.jsx]: Please add some inline comment for this mystery number... What happens if it's not 2?

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/service.js [hunk 3]
```diff
@@ -40,6 +40,11 @@ const nextSlide = (currentSlideNum, numberOfSlides, podId) => {
   }
 };
 
+const zoomSlide = (currentSlideNum, podId, widthRation, HeightRatio, xOffset, yOffset) => {
+  makeCall('zoomSlide', currentSlideNum, podId, widthRation, HeightRatio, xOffset, yOffset);
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/service.js]: widthRatio
heightRatio

File: bigbluebutton-html5/public/fonts/BbbIcons/bbb-icons.svg [hunk 4]
```diff
@@ -332,10 +332,13 @@ c0 -3 0 -7 -1 -10l-46 -269l242 126c8 3 18 5 28 5s20 -2 28 -6z" />
     <glyph glyph-name="star_filled" unicode="&#xe952;" 
 d="M243 -167c-33 1 -60 28 -60 61c0 2 1 7 1 10l46 266l-196 192c-10 9 -18 29 -18 42c0 29 24 56 52 60l269 39l121 244c9 19 33 34 54 34s45 -15 54 -34l120 -243l270 -40c28 -4 52 -31 52 -60c0 -13 -8 -33 -18 -42l-196 -192l46 -268c0 -3 1 -7 1 -10
 c0 -17 -11 -39 -25 -49c-9 -6 -24 -11 -35 -11c-8 0 -21 3 -28 7l-241 127l-241 -127c-7 -3 -19 -6 -27 -6h-1z" />
+<<<<<<< HEAD
```
Comment (antobinary) [inline bigbluebutton-html5/public/fonts/BbbIcons/bbb-icons.svg]: Merge conflict, please resolve

**Comment by jibon57** [comment]
@Tainan404 , thanks for great work.  Will you please share they way so that I can test it in my server?

**Comment by ffdixon** [comment]
This is part of the upcoming BigBlueButton 2.2 release.  This release is still under development.  We'll post updates to bigbluebutton-dev when a beta release is ready for others to try out.

**Comment by nekapoor** [comment]
@Tainan404 does this commit include a fit to width option for uploaded documents?

**Comment by capilkey** [comment]
@nekapoor Fit-to-width is included in this follow up PR https://github.com/bigbluebutton/bigbluebutton/pull/6104

**Comment by nekapoor** [comment]
@capilkey awesome and as Fred stated, there is no way to test this out until the 2.2 release?

**Comment by capilkey** [comment]
You could maybe take a 2.0 server and then run every part of BBB from source on the master branch, but I wouldn't recommend it. You'd need at least 10 terminals and even then you might still be missing config files.

**Comment by ffdixon** [comment]
Hi Nekapoor, this is part of BigBlueButton 2.2, which is still in development.  We look forward to getting to BigBlueButton 2.2-beta soon so you can try it out as well.

**Pull request merged**